swagger/swagger.yaml

```diff
@@ -1059,21 +1058,6 @@ paths:
           schema:
             $ref: '#/definitions/schema.OpenAIResponse'
       summary: Generate a chat completions for a given prompt and model.
-  /v1/completions:
```

Comment:
@dave-gray101 this is quite weird, swagger seems confused and doesn't see anymore the completions endpoint

Reply:
reason found: it was a space between the comment confusing swagger :sweat_smile: . Fixed in https://github.com/mudler/LocalAI/pull/4805